content/source/docs/cloud/integrations/service-now/index.html.md

```diff
@@ -2,192 +2,207 @@
+![Screenshot: ServiceNow Store Page](./images/service-now-store.png "Screenshot of the ServiceNow Store listing for the Terraform Integration")
 
-##### Worker Poll Run State
+## Connect ServiceNow to Terraform Enterprise
 
-The worker synchronizes ServiceNow with the current run state of Terraform workspaces by polling the Terraform Enterprise API. On state changes, the worker adds a comment to the ServiceNow request item with the updated run state and other metadata.
+-> **ServiceNow Roles:** `admin` or `x_terraform.config_user`
 
-![screenshot: ServiceNow integration comments](./images/service-now-comments.png)
+Once the integration is installed, the ServiceNow Admin can connect your
+ServiceNow instance to Terraform Enterprise. Before you begin, you will need
+four pieces of information from your Terraform Enterprise Admin.
 
-## Connecting to Terraform Enterprise
+   1. The Organization Name.
+   1. The Team API Token.
+   1. The hostname of your Terraform Enterprise instance.
+   1. The OAuth Token ID(s) and repository identifier(s) for the version control
+      repository or repositories containing Terraform configuration to be
+      deployed via ServiceNow.
 
--> **Roles Required:** `admin` or `x_terraform.config_user`
+Once you have this information, connect ServiceNow to Terraform Enterprise with
+the following steps.
 
-1. From the ServiceNow Service Management Screen.
-1. Using the left-hand navigation, open the configuration table for the integration to manage the Terraform Enterprise connection.
+1. Navigate to your ServiceNow Service Management Screen.
+1. Using the left-hand navigation, open the configuration table for the
+   integration to manage the Terraform Enterprise connection.
     - Terraform > Configs
-1. Click on "New" to create a new Terraform Enterprise connection:
-    - Set API Team Token to the Terraform Enterprise [Team Token](../../users-teams-organizations/api-tokens.html#team-api-tokens) you created earlier.
-    - Set Hostname to the hostname of your Terraform Enterprise instance. (If you're using the SaaS version of Terraform Cloud, this is app.terraform.io.)
-    - Set Org Name to the name of the Terraform Enterprise organization you wish to use for new workspaces created by ServiceNow.
-
-## Adding the Terraform Service Catalog
-
-1. In ServiceNow, open the Service Catalog > Catalogs view by searching for "catalogs" in the left-hand navigation.
-   1. Click the plus sign in the top right.
-   1. Select Terraform and choose a place to add it.
-
-At this point, your users can request Terraform infrastructure via ServiceNow, but there are not yet any infrastructure items available to request.
+1. Click on "New" to create a new Terraform Enterprise connection.
+    - Set Org Name to the name of the Terraform Enterprise Organization Name.
+    - Click on the "Lock" icon to set Hostname to the hostname of your Terraform
+      Enterprise instance. If you are using the SaaS version of Terraform Cloud,
```

Comment:
```suggestion
      Cloud or Enterprise instance. If you are using the SaaS version of Terraform Cloud,
```

Reply:
I decided to include both here because the default may be confusing otherwise? Up to your discretion @robin-norwood